src/views/Events.vue

```diff
@@ -109,13 +109,14 @@ export default {
     },
 
     //unixToDate: converts the given unix timestamp to a javascript date number
+    //legacy code
     unixToDate(unix) {
-      return unix * 1000;
+      return unix;
     },
 
     //Helper function: determines if a given date is more than 24 hours in the past
     isPast(date) {
-      return this.unixToDate(date) <= Date.now() - 24 * 60 * 60 * 1000; //day in milliseconds
+      return new Date(date) <= Date.now();
```

Comment:
Update the comment to reflect what this method is doing

Reply:
Method still does the same thing, only adjusted for the different format.